playbooks/collect_activity/stats.py

```diff
@@ -52,8 +75,22 @@
 
     ms = modulestore()
 
+    start_date = args.start_date
+    end_date = args.end_date
+
+    if start_date is None or end_date is None:
+        # If a time interval is not passed as arguments
+        # get the active users for the last full month.
+        beginning_of_this_month = datetime.utcnow().date().replace(day=1)
+        end_of_last_month = beginning_of_this_month - timedelta(days=1)
+        beginning_of_last_month = end_of_last_month.replace(day=1)
+
+        start_date = beginning_of_last_month
+        end_date = end_of_last_month
+
     stats = {
         'users': User.objects.count(),
+        'active_users': User.objects.filter(last_login__range=(start_date, end_date)).count(),
```

Comment:
@janimo, this is a native datetime though the time zone support is active. You should use the UTC timezone for these datetime objects.

Reply:
@lgp171188 you mean in the active_users filter? That is handled differently from the other date queries (like set of instances created in last month) because they run on the instances database and not on OCIM's own?

Is using utcnow() above not enough? Should it be datetime.now(tz=utc)? From what I see the last_login is stored in UTC, and so is I assume the time argument passed to the script.

Reply:
> The return value will give us the current date in UTC, right?

@janimo, a time zone has a meaning only for the `datetime` objects and not `date` objects.

Reply:
@janimo, this is still not addressed.

Reply:
@lgp171188 I have fixed it to use tz aware datetimes instead.